docs/pages/blog/embracing-the-future.md

```diff
@@ -0,0 +1,128 @@
+
+#### Establish a Signer Connection:
+
+```typescript
+import * as IdentityKit from '@nfid/identity-kit'
+import { NFID, II } from '@nfid/identity-kit/adapter'
+
+const nfid = NFID.config({
+  providerUrl: 'https://nfid.one',
+})
+const ii = II.config({
+  providerUrl: 'https://identity.ic0.app',
+})
+
+IdentityKit.config({
+  derivationOrigin: 'https://<canisterId>.ic0.app',
+  appMeta: {
+    name: 'My App',
+    logo: 'https://domain.com/path/to/logo.png',
+  },
+})
+
+// Establish a connection with the Signer the user has selected:
+const connection = await IdentityKit.connect({
+  signer: nfid,
+})
+```
+
+#### Exploring available standards on a given signer:
+
+```typescript
+type Response = string[]
+
+const supported: SupportedStandards = await connection.supportedStandards()
+// ["ICRC-31", "ICRC-32"]
+```
+
+### ICRC-31 Get Principals
+
+The `icrc31_get_principals` message plays a crucial role in identity management. It allows applications to retrieve information about the identities managed by a signer, streamlining the authentication process. This standard is pivotal in ensuring that users can manage their digital identities efficiently and securely.
+
+```typescript
+type Response = {
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-32 Sign Challenge
+
+The `icrc32_sign_challenge` method provides a secure mechanism for verifying ownership of user identities. By facilitating cryptographic proofs, this standard strengthens the security framework of applications, ensuring that user identities are protected and authenticated reliably.
+
+```typescript
+type Response = {
+  version: string // "1"
+  principals: string[] // ["gyu2j-2ni7o-o6yjt-n7lyh-x3sxq-zh7hp-sjvqe-t7oul-4eehb-2gvtt-Jae"]
+}
+
+const response: Response = await connection.getPrincipals()
+```
+
+### ICRC-33 Call Canister
+
+This standard allows applications to request the execution of calls to third-party canisters by the signer. By using the `icrc33_call_canister` method, applications can interact with a broader range of services and functionalities, enhancing the versatility and reach of ICP-based applications.
+
+```typescript
+type Response = {
+  version: string
+  signedChallenge: {
+    publicKey: string
+    signature: string
+  }
+}
+const response: Response = await connection.signChallange({
+  challenge: 'UjwgsORvEzp98TmB1cAIseNOoD9+GLyN/1DzJ5+jxZM=',
+})
+```
+
+### ICRC-34 Get Delegation
+
+The `icrc34_get_delegation` method is designed to facilitate the delegation of identity management. This process allows users to securely manage their identities and permissions, providing a flexible and secure framework for identity delegation within the ICP ecosystem.
+
+```typescript
+const response = await connection.getDelegation({...})
+```
+
+## Conclusion
```

Comment:
I would still mention a few more things:
* We're in the process of finalizing the standards. Now would be a great time to take a look and provide feedback.
* This is a community wide effort. While NFID spearheads the effort and provides the solid foundation to build this identity kit, everybody is welcome to contribute and help speed things up.
* There are also a lot of surrounding work that could be contributed (e.g. the metadata registry), pushing forward in making sure all the relevant infrastructure supports ICRC-21 (might benefit from library support as well), etc...

Reply:
Thanks @frederikrothenberger, these are really good points! I'm working on integrating all of this 🙏